src/azure-cli/azure/cli/command_modules/iot/_help.py

```diff
@@ -968,3 +971,70 @@
 type: command
 short-summary: Update metadata for an IoT Central application.
 """
+
+helps['iot hub private-endpoint-connection'] = """
+type: group
+short-summary: Manage private endpoint connections to an IoT Hub.
+"""
+
+helps['iot hub private-endpoint-connection approve'] = """
```

Comment:
Please implement private endpoint connection and private link resource command under network. You can simply register the type, api-version and add corresponding tests. 
https://github.com/Azure/azure-cli/blob/fe18bc1c9832da8f0d1dc9082d14bbcd800926e5/src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/custom.py#L15-L22

Reply:
our official doc is a little bit out-of-date. We will change it later. If you have any question, please reach out to me.